packages/react/src/utils/__tests__/__snapshots__/publicAPI.test.js.snap

```diff
@@ -22964,14 +23015,11 @@ Map {
   },
   "Tab" => Object {
     "defaultProps": Object {
-      "href": "#",
```

Comment:
Looks like they are moving some defaults to other areas of the functional component. 
![image](https://user-images.githubusercontent.com/1590966/107974598-d5e73e00-6f84-11eb-99a5-86a5f1dd5f1c.png)